addons/product/models/product_attribute.py

```diff
@@ -246,9 +246,9 @@ class ProductTemplateAttributeExclusion(models.Model):
     _description = 'Product Template Attribute Exclusion'
 
     product_template_attribute_value_id = fields.Many2one(
-        'product.template.attribute.value', string="Attribute Value", ondelete='cascade')
+        'product.template.attribute.value', string="Attribute Value", ondelete='cascade', index=True)
```

Comment:
Is is acceptable to create Index for existing field in stable version ?

Reply:
I believe it is OK as long as there is a good reason (big perf impact) and we are not very far into the release. Not 100 percent sure though.